src/routes/examples/geojson_line_layer/+page.svelte

```diff
@@ -48,6 +48,7 @@
   standardControls
   center={[-68.137, 45.137]}
   zoom={5}
+  hash
```

Comment:
Could you also just add a sentence or two above the map mentioning that this page enables tracking the bounds in the hash?

Reply:
Never mind I got it :)